### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by tarsdev**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

Uploaded my image for blog.

### CODE REVIEW

**Comment by oanarosca** [comment]
Thanks! Please check the boxes in the description and add your image file - `tdev.jpg` is not part of your commit.

**Comment by tarsdev** [comment]
I have done it

**Comment by oanarosca** [comment]
@tarsdev the .jpg file is not present -- look at the `Files changed` tab, you only edited the blogs file. Add the image and commit again.

File: _data/blogs.yml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Please add your entries to a random position in the file.
 # This is to remove conflicts with other people adding entries.
 
+- name: tarsdev
+  blog: https://tarsdev.wordpress.com/
+  blog_img: tdev.jpg
```
Comment (robbyoconnor) [inline _data/blogs.yml:6]: this does not exist.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
see comments. add the image to `images/blogs`

**Comment by oanarosca** [comment]
Any updates on this?

**Comment by robbyoconnor** [comment]
Closing due to lack of activity.

**Pull request closed without merging**